sdk/eventhub/Azure.Messaging.EventHubs/src/Primitives/EventProcessor.cs

```diff
@@ -1588,12 +1593,12 @@ private async Task RunProcessingAsync(CancellationToken cancellationToken)
                         _ = InvokeOnProcessingErrorAsync(slowException, null, Resources.OperationEventProcessingLoop, CancellationToken.None);
                     }
 
-                    // If the number of partitions owned is maximum advisable set, emit a warning.  This is an inaccurate heuristic and does
-                    // not apply to all workloads.  The error handler will not be pinged to avoid false positive notifications.
+                    // If the number of partitions owned has changed since the cycle started and is above maximum advisable set, emit a warning.  This is
+                    // an inaccurate heuristic and does not apply to all workloads.  The error handler will not be pinged to avoid false positive notifications.
```

Comment:
```suggestion
                     // If the number of partitions owned has changed since the cycle started and is above maximum advisable set, emit a warning.  This is
                    // a loose heuristic and does not apply to all workloads.  The error handler will not be pinged to avoid false positive notifications.
```